lib/web/mage/math.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+(function (root, factory) {
+    'use strict';
+
+    if (typeof define === 'function' && define.amd) {
+        define([
+            'jquery'
+        ], factory);
+    } else {
+        factory(root.jQuery);
+    }
+}(this, function ($, mage) {
+    'use strict';
```

Comment:
What's the motivation for having this be a UMD module in `lib/web` and not a normal AMD module?

Reply:
@DrewML I've proceeded the exact way how the `mage/translate` is added to the `mage`. Should I change the approach here, move the file to other folder? 🤔 What will you recommend?
Thank you!

Reply:
Hey @vasilii-b - thanks for being open to the feedback! Asked because I wasn't sure if there was something I was overlooking or not.

I think we'd be better off writing this as an AMD module (aka `define([], function() {})`) in `mage/utils`. Examples: https://github.com/magento/magento2/tree/97fd70faebc7d4ff15c00ec8f018e30d948f47c5/lib/web/mage/utils